### DISCUSSION TITLE
Error when converting the waymo dataset

### DISCUSSION THREAD

**Discussion by JiSuanJiDaWang** [open-mmlab/mmdetection3d#2023] [Q&A] [answered]
Hi, I encountered the following errors when converting waymo dataset on version 1.1.
![img_v2_62703f0e-ca4c-44a9-9b82-15994e525e5g](https://user-images.githubusercontent.com/82489384/201298991-7d185079-0d89-4596-80aa-e70154b525ed.png)

**Comment by JingweiZhang12** [comment, marked as answer]
This parsing is for waymo1.4. Suggest using the waymo1.4. It add a `seg_labels`.

**Reply by JiSuanJiDaWang** [reply]
Thanks. Do we have other options? I mean modify the code or something...

**Reply by JingweiZhang12** [reply]
Just remove the `seg_labesl` variable.
